From 42bbb94b54acade99b1e6811d7fdfadef0833a9c Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Sun, 30 Apr 2017 11:34:23 +0800
Subject: [PATCH 63/69] arm64: allwinner: a64: enable display-related
 regulators for Pinebook

Pinebook's LCD display relies on some regulators.

Enable them.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts
index 8cb8bb7ab2dc..1af14de5ad66 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts
@@ -236,6 +236,12 @@
 	regulator-name = "vcc-rtc";
 };
 
+&simplefb_lcd {
+	vcc-lcd-supply = <&reg_dc1sw>;
+	edp-2v5-supply = <&reg_dldo2>;
+	edp-1v2-supply = <&reg_fldo1>;
+};
+
 &uart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart0_pins_a>;
-- 
2.12.2

